portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,7 +51,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String newComment = request.getParameter("comment");
     if (!newComment.isEmpty()) {
-      messages.add(newComment);
+      Entity commentEntity = new Entity("Comments");
+      commentEntity.setProperty("comment", newComment);
+      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
do you need to call getDatastoreService() every time? or can the datastore be reused?

Reply:
They called it in both GET and POST in the tutorials. I will try making it a constant for the class.

Reply:
I tried using a global variable for the datastore. It worked most of the time, but there would be instances when it would not retrieve the comment I just added from the datastore.